### PR TITLE
Fibo 236

### PR DESCRIPTION
**Pull request by Karpas**
New window for graphs display on FIBO

### CODE REVIEW

File: fibo/src/components/VisNetwork.vue [hunk 1]
```diff
@@ -43,17 +43,67 @@
...
+      v-bind:class="{ fade: !modalVisible }"
+    >
+      <div
+        class="modal-dialog-full-width modal-dialog momodel modal-fluid"
+        role="document"
+      >
+        <div class="modal-content-full-width modal-content">
+          <div class="modal-header-full-width modal-header">
+            <h5 class="modal-title w-100" id="graphModalLabel">
+              <b class="mr-1 ml-4">Data model for "<i>${clazz.label}</i>"</b>
```
Comment (przemekgradzki) [inline fibo/src/components/VisNetwork.vue:77]: On page [BE/Corporations/Corporations/BoardAgreement](http://pr-90.spec.edmcouncil.org/fibo/ontology/BE/Corporations/Corporations/BoardAgreement?version=master%2F2021Q2) I clicked "expand...." and a window appeared with _Data model for "**${clazz.label}**"_
image

**Review by przemekgradzki — CHANGES_REQUESTED** [review]

File: fibo/src/components/VisNetwork.vue (hunk 1 above)
Comment (mereolog) [reply fibo/src/components/VisNetwork.vue:77]: Also @Karpas could you make 'expand' button more prominent/visible and more self-explanatory, e.g. "Expand diagram" or "Diagram in a new window" or something like that?

**Review by mereolog — CHANGES_REQUESTED** [review]
Also @Karpas could you make 'expand' button more prominent/visible and more self-explanatory, e.g. "Expand diagram" or "Diagram in a new window" or something like that?

**Comment by mereolog** [comment]
Looks much better.
One more thing - could you move the Expand button to the bottom of the initial diagram and center it in the middle?
image

**Comment by Karpas** [comment]
Done.

**Comment by mereolog** [comment]
@Karpas No, this is not done.
I asked for a move not  a copy:
image
The current display looks preposterous.

**Review by mereolog — APPROVED** [review]

**Review by dasawanaka — APPROVED** [review]
Approved 👍

**Pull request merged**